src/app/styles/navbar/navbar.module.css

```diff
@@ -12,13 +12,17 @@
     align-items: center;
     height: 100%;
     position: relative;
+    font-family: "Lato";
+    font-weight: 600;
+    max-width: 100%;
 }
 
 .navbar {
     height: 60px;
     background-color: #2b2b33;
-    position: relative;
+    position: sticky;
```

Comment:
not sure if its been fixed, but as of now mobile view if you click on empty space the navbar will not close, which kinda sucks, but tbf i don't know how many times they will be using mobile.